esrally/track/track.py

```diff
@@ -910,26 +910,26 @@ def ignore_response_error_level(self):
                 ignore_response_error_level not in Task.IGNORE_RESPONSE_ERROR_LEVEL_WHITELIST:
             raise exceptions.InvalidSyntax(
                 f"Task [{self}] specifies ignore-response-error-level to [{ignore_response_error_level}] but "
-                f"the only allowed values are [{Task.IGNORE_RESPONSE_ERROR_LEVEL_WHITELIST}].")
+                f'the only allowed values are [{",".join(Task.IGNORE_RESPONSE_ERROR_LEVEL_WHITELIST)}].')
```

Comment:
Super-nit: I tend to use single-quotes inside f-strings (i.e. for the `join`) but I'm fine with this as well.